api/exclusions/exclusionService.ts

```diff
@@ -31,8 +31,10 @@ export async function confirmUserExclusion(req: EnhancedRequest, res: Response,
       userType: 'Judicial',
     },
   ];
-  if (['400, 401, 402, 403, 500, 503'].indexOf(req.body.exclusionDescription)) {
-    return res.status(req.body.exclusionDescription).send('error: {status: req.body.exclusionDescription}"');
+  const errorCodes: string[] = ['400', '401', '402', '403', '500', '503'];
+  const value: string = req.body.exclusionDescription;
+  if (errorCodes.indexOf(value) !== -1) {
+    return res.status(parseInt(value, 10)).send('error: {status: req.body.exclusionDescription}"');
```

Comment:
Caught an issue with this on tests - req.body.exclusionDescription is being shown as string - value should be being passed into string

Reply:
good spot, this was originally from Paul H, I am not aware this template is doing nothing. Changed now.